suite-common/suite-config/src/features.ts

```diff
@@ -6,12 +6,13 @@
 // General flags
 export const FLAGS = {
     GOOGLE_DRIVE_SYNC: true, // Google Drive sync (used for labeling)
-    RBF: true, // replace by fee feature in the send form
+    DROPBOX_SYNC: true, // TODO ADD
+    RBF: true, // TODO REMOVE replace by fee feature in the send form
     FILE_SYSTEM_SYNC: false, // File system sync (used for labeling)
-    ONION_LOCATION_META: true, // Show TOR onion-location meta tag in page head
-    EXPORT_TRANSACTIONS: true, // Display export option on transactions
-    SEARCH_TRANSACTIONS: true, // Display search option on transactions
-    DESKTOP_AUTO_UPDATER: true, // Runs auto updater code on desktop
+    ONION_LOCATION_META: true, // TODO MAYBE REMOVE vyzkoumat k čemu to je Show TOR onion-location meta tag in page head
```

Comment:
It is for meta tag in html head. If you visit suite.trezor.io from tor browser, it should offer you onion address. I think we can have it enabled by default without feature flag